bundle/bin/rke2-uninstall.ps1

```diff
@@ -167,29 +155,52 @@ Get-Service -Name "rke2" -ErrorAction Ignore | Where-Object {$_.Status -eq "Runn
             sc.exe delete rke2
         }
     }
+}
 
+# clean up wins service
+Get-Service -Name "wins" -ErrorAction Ignore | Where-Object {$_.Status -eq "Running"} | ForEach-Object {
```

Comment:
You may want to consider this approach. https://github.com/rancher/system-agent-installer-rke2/blob/main/package/run.ps1#L87-L95